operations/helm/charts/mimir-distributed/values.yaml

```
@@ -990,133 +990,349 @@ compactor:
   extraEnvFrom: []
 
 memcached:
+  # -- Specifies whether memcached should be enabled
   enabled: false
-  architecture: high-availability
-  arguments:
-    - -m 8192
-    - -o
-    - modern
-    - -v
-    - -I 1m
-    - -c 4096
+
+  # -- Total number of memcached replicas
+  replicas: 1
+
+  # -- Container image settings for memcached
   image:
     repository: memcached
-    tag: 1.6.9
-  # maxItemMemory is in bytes. Should match memcached -I flag (which is in MB)
-  # It is a string to avoid https://github.com/helm/helm/issues/1707.
-  maxItemMemory: '1048576'  # (* 1 (* 1024 1024))
-  metrics:
-    enabled: true
-    image:
-      registry: quay.io
-      repository: prometheus/memcached-exporter
-      tag: v0.9.0
-  replicaCount: 1
+    tag: 1.6.9-alpine
+    pullPolicy: IfNotPresent
+
```

this is being used in the template but is not here

```suggestion
    pullSecrets: []

```